safeweb/http.go

```diff
@@ -0,0 +1,89 @@
+package safeweb
```

Comment:
nit: we'll want detailed docs for the package once it's closer to merge

Reply:
:100: waiting on this taking its final form to write the longer documentation.

Reply:
Added some longer form documentation including example usage.